firewall.rb

```diff
@@ -17,7 +17,8 @@
     shell "ufw allow in on eth1 proto tcp from 10.30.0.0/15 to any port 5432"
 
     # Allow collectd connections from docker.
-    shell "ufw allow in on docker_gwbridge proto any from 172.16.0.0/12 to any port 8125"
+    shell "ufw allow in on docker_gwbridge proto udp from 172.16.0.0/12 to any port 8125"
```

Comment:
So the docker apps will send out stats to a `collectd` instance running in their host on these 2 ports? UDP 8125 for Librato, TCP 8126 for DataDog?

Reply:
After chatting with @nickbrowne, I believe I need to allow both UDP and TCP on 8126 for now. This is because we have the datadog agent running on UDP port 8126 (normally it's on 8125).

Reply:
Yep, normally statsd runs on UDP 8125, but we're actually running two instances of it (the second on 8126) while we transition everything across. One is still funneling things into librato through collectd, the other is going to the datadog agent.